easybuild/easyconfigs/q/QCMaquis/QCMaquis-4.0.0-iomkl-2023a.eb

```diff
@@ -29,6 +29,19 @@ configopts = '-DBUILD_OPENMOLCAS_INTERFACE=ON -DCMAKE_Fortran_FLAGS="$FFLAGS -fp
 
 runtest = True
 
+# Fix OpenMolcas 25.06 installation with DMRG
```

Comment:
@pavelToman please include link to issue you opened upstream to report these problems in a comment